src/components/operations/operation-details/ko/runtime/operation-details.ts

```diff
@@ -329,6 +334,40 @@ export class OperationDetails {
         return result;
     }
 
+    private parseExamples(operation: Operation): void {
+        const examples = operation.getMeaningfulResponses().reduce((acc, cur) => {
+            const representations = cur.meaningfulRepresentations();
+            if (!representations || !representations.length) return acc;
+
+            const examplesObj = {}
+            representations.forEach(representation => {
+                const value = representation.examples?.[0]?.value;
+                if (!value) return;
+
+                let valueObj;
+                try {
+                    valueObj = JSON.parse(value);
+                } catch (e) {
+                    return;
+                }
+
+                const contentTypeObj = {}
+                Object.entries(valueObj).forEach(([key, val]) => {
+                    if (typeof val === 'object') return
```

Comment:
his return will break the for loop, lets say your first element is not object type but the second one is object, the second one will not be seen

Reply:
No, it only stops the current iteration:
![image](https://user-images.githubusercontent.com/92859870/204582148-7dee1e57-5416-4b39-b3b0-41d5e4a8aa62.png)